---
package: "lucia-auth"
type: "patch"
---

Fix use `reject` instead of throwing error in `validate`